src/com/gfredericks/test/chuck/generators.cljc

```diff
@@ -238,8 +238,8 @@
      (scalb (core/double signed-significand) (core/int exp)))
    (gen/tuple
     (let [bignumber (apply * (repeat 52 2))]
-      (bounded-int (- bignumber) bignumber))
-    (bounded-int -1022 1023))))
+      (gen/large-integer* {:min (- bignumber) :max bignumber}))
+    (gen/large-integer* {:min -1022 :max 1023}))))
```

Comment:
actually I entirely misread this, they **are** equivalent (I was comparing the new line 242 with the old line 241 or something, it was dumb).